admin/Makefile

```diff
@@ -0,0 +1,39 @@
+BUILD_DIR?=.build
+VENV?=${BUILD_DIR}/venv
+
+.PHONY: install
+install: ${BUILD_DIR}/venv.timestamp \
+	${BUILD_DIR}/commons.timestamp \
+	${BUILD_DIR}/requirements.timestamp
+
+.PHONY: init_db
+init_db:
+	#Configure the database:
+	${VENV}/bin/initialize_db_main development.ini
+
+.PHONY: test
+test:
+	#Run your project's tests.
+	${VENV}/bin/pytest
+
+.PHONY: serve
+serve:
+	#Run your project.
+	${VENV}/bin/pserve development.ini
+
+${BUILD_DIR}/venv.timestamp:
+	#Create a Python virtual environment.
+	python3 -m venv ${VENV}
+	#Upgrade packaging tools.
+	${VENV}/bin/pip install --upgrade pip setuptools
```

Comment:
Relay risky ...

Reply:
Why ?

Reply:
We get any last version of pip and setuptools, and nearly half of setuptools versions is buggy ...

Reply:
You mean you prefer if we set package versions here ?

Reply:
Yes